### ISSUE TITLE
SSLError([SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:852))

### ISSUE THREAD

**Issue by munachisonwadike** [asyml/forte#681]
Hello!

**Issue**
When I try to run the code for the 3-part Forte Tutorial, involving elasticsearch, I get an error regarding SSL versions.
The stacktrace is quite long, so I have attached screenshots from the top and bottom parts.
I found some pages which suggested that the solution may be to disable SSL [[link](https://github.com/elastic/elasticsearch-py/issues/712#issuecomment-357750570)] or set `verify_certs=False` [[link](https://github.com/elastic/elasticsearch-py/issues/712#issuecomment-362713658)].
However, some other pages seem to suggest the problem is I need to downgrade the `urllib3` version [[link](https://stackoverflow.com/questions/64960430/python-requests-with-proxy-results-in-sslerror-wrong-version-number/66656188#66656188)].
I was hoping to inquire as to what may the recommended best practice? And/or what we consider to be the correct SSL version?
Thank you so much~ 🙏🏾 
Warm regards,

**Screenshots**
_a) From the top part of the stacktrace_

![image](https://user-images.githubusercontent.com/22077758/158058118-7c04cbb0-9820-48ca-862a-725163239586.png)

_b) From the bottom part of the stacktrace_
![image](https://user-images.githubusercontent.com/22077758/158057972-e6b9069d-5d6f-4d4b-b1aa-4c274fd88bd1.png)

**Comment by hunterhector** [comment]
maybe @J007X  could help take a look

**Comment by munachisonwadike** [comment]
Hello! @hunterhector @J007X. I think I did managed to get to the root of this issue!
I noticed from this [[post](https://stackoverflow.com/questions/65516325/ssl-wrong-version-number-on-python-request)] that the error I was having comes from a bug in urllib3 is such that we need to change https to http.
As per the tutorial, we are supposed to pass a dictionary called elastic_search_config.
Here, the host is to be specified as `localhost:9200`, however,  I was using `https://localhost:9200`
It works if I either change to `http`, or remove the `https://` entirely!
Warm regards,
